docs/content/configuration/csi_ug_config_create_storageclasses.md

```diff
@@ -22,6 +24,13 @@ _<a name=spaceefficiency>**Table:**</a> `SpaceEfficiency` parameter definitions
 
 - The IBM DS8000 Family `pool` value is the pool ID and not the pool name as is used in other storage systems.
 - Be sure that the `pool` value is the name of an existing pool on the storage system.
+- To create a volume with HyperSwap on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
```

Comment:
@ArbelNathan do we still plan to include HyperSwap as part of 1.7.0?

Reply:
@oriyarde  i'm waiting for content confirmation in CSI-3242 from @ArbelNathan  before removing all HyperSwap references.

Reply:
I commented there

Reply:
Thanks - I've now removed the information from the latest version
DONE